modelscope/hub/repository.py

```diff
@@ -105,7 +105,7 @@ def add_lfs_type(self, file_name_suffix: str):
                 examples '*.safetensors'
         """
         os.system(
-            "printf '%s filter=lfs diff=lfs merge=lfs -text\n'>>%s" %
+            "printf '\n%s filter=lfs diff=lfs merge=lfs -text\n'>>%s" %
```

Comment:
why add '\n'  at the begin of line？

Reply:
The default `.gitattributes` doesn't have a empty line at the end, if `\n` is not added, it will be directly appended after the last line.